src/graphql/admin/types/object/user.ts

```diff
@@ -1,29 +1,15 @@
-import { Accounts } from "@app"
-import { mapError } from "@graphql/error-map"
 import { GT } from "@graphql/index"
 import Language from "@graphql/types/scalar/language"
 import Phone from "@graphql/types/scalar/phone"
 import Timestamp from "@graphql/types/scalar/timestamp"
 
-import Account from "./account"
-
 const User = GT.Object<IdentityPhone>({
   name: "User",
 
   fields: () => ({
     id: { type: GT.NonNullID },
     phone: { type: GT.NonNull(Phone) },
     language: { type: GT.NonNull(Language) },
-    defaultAccount: {
```

Comment:
is it not used?

Reply:
Admin panel never resolves from user -> account. Only account -> user.
Checked here https://github.com/GaloyMoney/admin-panel/tree/main/graphql